lib/facter/cve_2015_0235.rb

```diff
@@ -0,0 +1,27 @@
+# http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-0235
+# GHOST
+#
+# Tests for an unpatched rpm of glibc
+# More https://access.redhat.com/articles/1332213
+
+glibc_nvr = Facter::Util::Resolution.exec('rpm -q --qf "%{name}-%{version}-%{release}.%{arch}\n" glibc').scan(/^(glibc-(\d+\.\d+).*)$/)
+
+if glibc_nvr.length
+  Facter.add('cve_2015_0235') do
+    require 'puppet'
```

Comment:
Believe you need this after the Facter.add so it only runs on EL systems which should have rpm
```ruby
    confine :osfamily => 'RedHat'
```